terracumber_config/tf_files/tfvars/PR-TEST-manager43.tfvars

```diff
@@ -0,0 +1,14 @@
+############ Varaibles unique to spacewalk ###########
+
+IMAGE                  = "sles15sp4o"
+GIT_PROFILES_REPO      = "https://github.com/SUSE/spacewalk.git#:testsuite/features/profiles/internal_prv"
```

Comment:
For that variable, we will need to diverge it depending of the environment is placed in PRV or NUE.